packages/json-schema/index.js

```diff
@@ -0,0 +1,9 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/json-schema
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+const nodeMajorVersion = +process.versions.node.split('.')[0];
+module.exports = nodeMajorVersion >= 7 ?
+  require('./dist/src') :
+  require('./dist6/src');
```

Comment:
If we are supporting Node.js `>=8`, then there should be no `dist6` and we should always forward to `./dist/src`.